mirgecom/eos.py

```diff
@@ -58,6 +79,18 @@ class EOSDependentVars:
 
     temperature: np.ndarray
     pressure: np.ndarray
+    speed_of_sound: np.ndarray
```

Comment:
```suggestion
    speed_of_sound: Union[float, DOFArray]
``` 
`from typing import Union`

Reply:
(42f0bd5)

Reply:
Had trouble with this one.  Doc infrastructure hated this.